ts/Extensions/Annotations/Popup.ts

```diff
@@ -183,7 +183,7 @@ wrap(Pointer.prototype, 'onContainerMouseDown', function (this: Pointer, proceed
     }
 });
 
-H.Popup = function (this: Highcharts.Popup, parentDiv: HTMLDOMElement, iconsURL: string, chart: Chart): void {
+H.Popup = function (this: Highcharts.Popup, parentDiv: HTMLDOMElement, iconsURL: string, chart: Chart|undefined): void {
```

Comment:
There are so many `chart` references in the code, are you sure it's safe? For example `chart.plotTop`, `chart.series` etc. 

It seems `this.chart` is not defined in Class, so defining this as `chart?: Chart` should be a good idea to review this all

Reply:
Makes sense, we can replace the `chart` with the optional `chart`

Reply:
Done